package.json

```diff
@@ -12,8 +12,9 @@
     "postinstall": "npm prune && npm run build",
     "prepush": "opt --out prepush --exec 'npm run validate'",
     "start": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback --open",
-    "test": "NODE_ENV=test karma start",
-    "test:watch": "npm test -- --auto-watch --no-single-run --reporters progress",
+    "test": "jest",
+    "test:watch": "npm test -- --verbose --notify --watch",
```

Comment:
You can override with `npm run test:watch -- --verbose=false`, of course the same is true in the other direction. We just need to decide what to default to.